resources/views/partials/book-card.blade.php

```diff
@@ -1,6 +1,6 @@
 <div class="book-card">
 	<div class="book-cover">
-		<img src="{{ $book->cover }}" alt="{{ sprintf(__('%s book cover', 'pressbooks-network-catalog'), $book->title) }}" />
+		<a href="{{ $book->url }}" aria-hidden="true" tabindex="-1"><img src="{{ $book->cover }}" alt="{{ sprintf(__('%s book cover', 'pressbooks-network-catalog'), $book->title) }}" /></a>
```

Comment:
Wrap book cover in anchor element that links to book home page and hide it from screenreaders so that users of AT don't have a duplicate link announced (the book heading itself follows just after and also links to the book homepage)